app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,10 +6,23 @@ def index
   end
 
   def create
-    # binding.pry
   end
 
   def show
     @error = current_site.issues.where('issues.id = ?', params[:id]).first
   end
+
+   def update
+    @error = Issue.find(params[:id])
+    if @error.update_attributes(error_params)
+      Website.find(@error.website_id).subscribers.each do |member|
+        UserMailer.issue_solved(@error, member).deliver_now
+      end
```

Comment:
asta nu e corect. Nu vrem sa trimitem mailuri la useri cand se rezolva eroarea. O sa ne trebuiasca o noua metoda jos `def message_subscribers` in care o sa bagam ceva de genu cu un mesaj custom pe care il scrii in textarea aia. Plus ca o sa vrem la un moment dat sa facem resolve cand se trimite mesajul dar niciodata nu vrem sa trimitem mailuri cand se face un resolve normal. Nu e obligatoriu ca userii sa fie notificati cand se rezolva ceva.